aws/resource_aws_iam_group_policy.go

```diff
@@ -105,17 +117,22 @@ func resourceAwsIamGroupPolicyRead(d *schema.ResourceData, meta interface{}) err
 		return err
 	}
 
-	d.Set("group", group)
-	d.Set("name", name)
-	d.Set("policy", policy)
-
-	return nil
+	if err := d.Set("policy", policy); err != nil {
+		return err
+	}
+	if err := d.Set("name", name); err != nil {
+		return err
+	}
+	return d.Set("group", group)
```

Comment:
Same here regarding error context

```suggestion
	if err := d.Set("group", group); err != nil {
		return fmt.Errorf("error setting group: %s", err)
	}
	return nil
```

Reply:
Done.